pkgs/development/python-modules/simple-term-menu/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pyproject = true;
+  nativeBuildInputs = [ setuptools ];
+
+  disabled = pythonOlder "3.4";
+
+  pname = "simple-term-menu";
+  version = "1.6.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "be9c5dbd8df12a404b14cd8e95d6fc02d58c60e2555f65ddde41777c487fb3b9";
+  };
+
+  propagatedBuildInputs = [ ];
```

Comment:
```suggestion
``` 
If the package has no dependencies, we don't need to define `propagatedBuildInputs`.